tests/manage/z_cluster/test_ceph_pg_log_dups_trim.py

```diff
@@ -0,0 +1,111 @@
+        2) COT dups trim command is indeed present on the ODF consumed ceph version and is working
+        3) When the number of dups exceeds 6000 limit, an online dups trim should automatically
+        happen and it should come down to the default tracked number of osd pg log dups which is
+        3000 in ODF 4.11
+        """
+        # Create PVC and write some IO
+        pvc_obj1 = pvc_factory(interface=constants.CEPHBLOCKPOOL, size="10")
+        pod_obj1 = pod_factory(pvc=pvc_obj1)
+        pod_obj1.run_io(size="5g", storage_type="fs")
+        pod_obj1.get_fio_results()
+        # Get a random pgid
+        random_pgid = random.choice(
+            get_specific_pool_pgid(constants.DEFAULT_CEPHBLOCKPOOL)
+        )
+        log.info(f"Selected pgid:{random_pgid} to inject dups")
+        # Set osds in maintenance mode for running ceph-objectstore commands
+        set_osd_maintenance_mode(get_osd_deployments())
+        # Inject Corrupted dups into the pg via COT
+        inject_corrupted_dups_into_pg_via_cot(get_osd_deployments(), pgid=random_pgid)
+        total_dups_before_trim_list = get_pg_log_dups_count_via_cot(
+            get_osd_deployments(), pgid=random_pgid
+        )
+        log.info(
+            f"Total number of dups before trim per OSD:{total_dups_before_trim_list}"
+        )
+        # Exit osd maintenance mode
+        exit_osd_maintenance_mode(get_osd_deployments())
+        # Create PVC and write some IO
+        pvc_obj2 = pvc_factory(interface=constants.CEPHBLOCKPOOL, size="10")
+        pod_obj2 = pod_factory(pvc=pvc_obj2)
+        pod_obj2.run_io(size="5g", storage_type="fs")
+        pod_obj2.get_fio_results()
+        # Checked for the expected dups message in the osd logs
+        expected_log = (
+            "num of dups exceeded 6000. You can be hit by THE DUPS BUG "
+            "https://tracker.ceph.com/issues/53729. Consider ceph-objectstore-tool --op trim-pg-log-dups"
+        )
+        for osd in get_osd_pods():
+            osd_pod_log = get_pod_logs(pod_name=osd.name, container="osd")
+            assert (
+                expected_log in osd_pod_log
+            ), f"Number of dups exceeded warning is not found in {osd.name} logs"
+        log.info("num of dups exceeded warning is generated on all osd pods")
+        # Set osds in maintenance mode for running ceph-objectstore commands
+        osd_dep_list_obj = get_osd_deployments()
+        default_osd_pg_log_dups = get_osd_pg_log_dups_tracked()
+        set_osd_maintenance_mode(osd_dep_list_obj)
+        total_dups_after_trim_list = get_pg_log_dups_count_via_cot(
+            osd_dep_list_obj, pgid=random_pgid
+        )
+        for per_osd_dups in total_dups_after_trim_list:
+            assert (
+                per_osd_dups == default_osd_pg_log_dups
+            ), "pg log dups are not trimmed to the default tracked value:3000"
+        log.info(
+            "All the osd's pg log dups are trimmed to the default tracked value: 3000 "
```

Comment:
Same as my previous comment - change the value `3000` to `{default_osd_pg_log_dups}`

Reply:
Done